back-end/courses/migrations/0002_alter_course_thumbnail_image_url.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 5.2.4 on 2025-11-25 07:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('courses', '0001_initial'),
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)